response_operations_ui/views/reporting_units.py

```diff
@@ -76,3 +80,20 @@ def map_region(region):
         region = "GB"
 
     return region
+
+
+def map_respondent_status(status):
+    if status == "CREATED":
+        status = "Created"
+    elif status == "ACTIVE":
+        status = "Active"
+    elif status == "PENDING":
+        status = "Pending"
+    elif status == "ENABLED":
+        status = "Enabled"
+    elif status == "DISABLED":
+        status = "Disabled"
+    elif status == "SUSPENDED":
+        status = "Suspended"
+
+    return status
```

Comment:
We could just have a return in each if statement rather than setting a variable each time